lib/std/elf.zig

```diff
@@ -305,6 +305,8 @@ pub const STT_ARM_16BIT = STT_HIPROC;
 pub const VER_FLG_BASE = 0x1;
 pub const VER_FLG_WEAK = 0x2;
 
+pub const ELF_MAGIC = "\x7fELF";
```

Comment:
This declaration is in a namespace called "elf" already, so the prefix is redundant. It should be called instead `MAGIC`.

Reply:
Donzo :)